### PR TITLE
STORM-1542: Remove profile action retry in case of non-zero exit code

### PR DESCRIPTION
**Pull request by abhishekagarwal87**
This fixes the infinite retry problem in profiling. As a follow up on the discussion in STORM-1542, I have created  STORM-1568.

### CODE REVIEW

**Comment by abhishekagarwal87** [comment]
@kishorvpatil can you review this?

**Comment by kishorvpatil** [comment]
LGTM. +1.

File: bin/flight.bash [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-JDKPATH="/home/y/share/yjava_jdk/java"
```
Comment (HeartSaVioR) [inline bin/flight.bash]: OMG... It shouldn't be included...

**Comment by HeartSaVioR** [comment]
+1

**Pull request merged**